examples/verif-manual/vm-014.py

```diff
@@ -0,0 +1,162 @@
+in an eccentrically compressed steel strut of length L. The cross-section is a channel
+with the dimensions shown in the diagram. The ends are pinned at the point of load application.
+The distance between the centroid and the back of the channel is e, and the compressive force F
+acts in the plane of the back of the channel and in the symmetry plane of the channel.
+
+"""
+
+import os
+
+from ansys.mapdl.core import launch_mapdl
+
+# Launch MAPDL with specified settings
+mapdl = launch_mapdl(loglevel="WARNING", print_com=True, remove_temp_dir_on_exit=True)
+
+# Clear any existing database
+mapdl.clear()
+
+# Set the ANSYS version
+mapdl.com("ANSYS MEDIA REL. 2022R2 (05/13/2022) REF. VERIF. MANUAL: REL. 2022R2")
+
+# Run the FINISH command to exists normally from a processor
+mapdl.finish()
+
+# Run the /VERIFY command for VM14
+mapdl.run("/VERIFY,VM14")
+
+# Set the title of the analysis
+mapdl.title("VM14 LARGE DEFLECTION ECCENTRIC COMPRESSION OF SLENDER COLUMN")
+
+# Enter the model creation preprocessor
+mapdl.prep7()
+
+# Provide reference information for the analysis
+"""
+The references for the analysis can be found here:
+- STR. OF MATL., TIMOSHENKO, PART 1, 3RD ED., PG. 263, PROB. 1
+"""
+
+# Define element type and properties
+mapdl.et(1, "BEAM188", "", "", 3)  # Element type BEAM188
+mapdl.sectype(1, "BEAM", "CHAN")  # Section type BEAM CHAN
+mapdl.secdata(2.26, 2.26, 8, 0.39, 0.39, 0.22)  # Section data
+mapdl.secoffset("USER", "", 0.6465)  # Section offset
+
+# Define material properties
+mapdl.mp("EX", 1, 30e6)  # Young's modulus
+mapdl.mp("PRXY", 1, 0.3)  # Poisson's ratio
+
+# Define nodes and element connectivity
+mapdl.n(1)  # Node 1
+mapdl.n(5, "", 60)  # Node 5 at 60 degrees
+
+# Generate additional nodes
+mapdl.fill()
+
+# Define element connectivity
+mapdl.e(1, 2)  # Element 1 with nodes 1 and 2
+
+# Generates elements from an existing pattern
+mapdl.egen(4, 1, 1)
+
+# Apply load and boundary conditions
+mapdl.d(1, "ALL")  # Fix all degrees of freedom for node 1
+mapdl.f(5, "FY", -4000)  # Apply a negative force FY to node 5
+mapdl.dsym("SYMM", "Z")  # Apply symmetry boundary condition in Z-direction
+
+# select all entities
+mapdl.allsel()
+# element plot
+mapdl.eplot()
+
+# Finish the pre-processing processor
+mapdl.finish()
+
+# Enter the solution processor to define solution controls
+mapdl.slashsolu()
+
+# Activate large deflections
+mapdl.nlgeom("ON")
+
+# Set convergence tolerances
+mapdl.cnvtol("F", "", 1e-4)
+mapdl.cnvtol("M", "", 1e-4)
+
+# Controls the solution printout
+mapdl.outpr("", "LAST")
+
+# Run non-interactive commands
+with mapdl.non_interactive:
+    mapdl.run("/OUT,SCRATCH")  # redirects solver output to a file named "SCRATCH"
+    mapdl.solve()  # starts a solution
+    mapdl.finish()  # exists solution processor
+
+    mapdl.post1()  # enter post-processing processor
+    mapdl.run("/OUT")  # redirects output to the default system output file
+    mapdl.gopr()  # reactivates suppressed printout
```

Comment:
This is not needed as we are directly print the output we need with PyMAPDL. It can be simplified with this: 

```suggestion
mapdl.solve()  # starts a solution
mapdl.finish()  # exists solution processor

mapdl.post1()  # enter post-processing processor
```

Reply:
we need outpr command to control results to write in *.rst file. I will make changes accordingly.